src/runtime/verifier.c

```diff
@@ -276,12 +262,18 @@ static inline int verify_string(const void *buf, uoffset_t end, uoffset_t base,
  */
 static inline int verify_vector(const void *buf, uoffset_t end, uoffset_t base, uoffset_t offset, uoffset_t elem_size, uint16_t align, uoffset_t max_count)
 {
-    uoffset_t n;
-
-    verify(check_aligned_header(end, base, offset, align), flatcc_verify_error_vector_header_out_of_range_or_unaligned);
+    verify(check_header(end, base, offset), flatcc_verify_error_vector_header_out_of_range_or_unaligned);
     base += offset;
-    n = read_uoffset(buf, base);
+
+    uoffset_t n = read_uoffset(buf, base);
     base += offset_size;
+
```

Comment:
Add comment like:
/* This is due to incorrect buffers from other builders than cannot easily be ignored. */